Training/LoginViewController.m

```diff
@@ -0,0 +1,152 @@
+//
+//  LoginViewController.m
+//  Training
+//
+//  Created by María Eugenia Sakuda on 2/23/15.
+//  Copyright (c) 2015 María Eugenia Sakuda. All rights reserved.
+//
+
+#import "LoginViewController.h"
+#import "UIView+Toast.h"
+#import "InformationAPI.h"
+#import "TTTAttributedLabel.h"
+
+@interface LoginViewController ()
+
+@property (weak, nonatomic) IBOutlet UITextField *emailText;
+@property (weak, nonatomic) IBOutlet UITextField *passwordText;
+@property (weak, nonatomic) IBOutlet UIButton *loginButton;
+//@property (weak, nonatomic) IBOutlet UILabel *termsAndConditionsLabel;
+@property (weak, nonatomic) IBOutlet TTTAttributedLabel *termsAndConditionsLabel;
+@property (weak, nonatomic) IBOutlet UIButton *termsAndConditionsButton;
+@property (weak, nonatomic) IBOutlet UIButton *signUpButton;
+
+
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    [self hideElements];
+    [self.view makeToastActivity];
+    
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+    if([defaults objectForKey:@"email"]){
+        [self performSegueWithIdentifier:@"tabBarSegue" sender:self];
+    }
+    
+    [self.view hideToastActivity];
+    [self showElements];
+    
+    [self.loginButton.layer setBorderWidth:1.0f];
+    [self.loginButton.layer setBorderColor:[UIColor blackColor].CGColor];
+    
+    //Hypervínculos label
+    //[self setLabel];
+    
+    [self setButtonAsLabelLink:self.termsAndConditionsButton];
+    
+    //Is pendent to check if it's already logged
+    // Do any additional setup after loading the view.
+    
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}*/
+
+
+- (IBAction)login:(id)sender {
+    NSString *toastMessage = @"";
+    
+    if(![self isValidEmail: self.emailText.text]){
+        toastMessage = @"Email no válido";
+    } else {
+        NSString *realPassword = [[InformationAPI getData].userData objectForKey:self.emailText.text];
+        if (![realPassword isEqualToString:self.passwordText.text])
+            toastMessage = @"La contraseña no es válida";
+    }
+
+    if([toastMessage isEqualToString:@""]){
```

Comment:
No esta bueno que la forma de validar si todo salio bien sea leyendo el string del toastMessage, esa es otra consecuencia de que el usuario haya ingresado bien las credenciales, no la causa de que todo salio bien.

Cuando hagas la validacion como te indique anteriormente esto va a salir mas natural, algo asi:

if (email and password are valid) then (voy al tabbar) else (muestro toast)